MSAL/IdentityCore

```diff
@@ -1 +1 @@
-Subproject commit 3e287dc4e38a465a53c7a18c00d117629ce07782
```

Comment:
This pull request does not update CHANGELOG.md.

Please consider if this change would be noticeable to a partner or user and either update CHANGELOG.md or resolve this conversation.
<!-- changelog-comment -->